t/data/tests/tests/boot.pm

```diff
@@ -50,6 +50,8 @@ END
     script_run "echo '924095f2cb4d622a8796de66a5e0a44a  text' > text.md5";
     assert_script_run 'md5sum -c text.md5';
 
+    my $out = script_output('mount');
+    die "mount does not show any mount points?" unless $out =~ /.*\/.*on/;
```

Comment:
well, sure you get the mount output, but not as lines

```
coolo@f84#openQA>echo $(mount)
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime) proc on /proc type proc (rw,nosuid,nodev,noexec,relatime) devtmpfs on /dev type devtmpfs (rw,nosuid,size=16422316k,nr_inodes=4105579,mode=755) securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime) tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev) devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000) tmpfs on /run type tmpfs (rw,nosuid,nodev,mode=755) tmpfs on /sys/fs/cgroup type tmpfs (ro,nosuid,nodev,noexec,mode=755) cgroup on /sys/fs/cgroup/unified type cgroup2 (rw,nosuid,nodev,noexec,relatime)

Reply:
I guess I have that fixed now with /…/s instead of /…/m